src/Hosting/Systemd/testassets/IntegrationTestApp/IntegrationTestApp.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IsPackable>true</IsPackable>
```

Comment:
Can you remove this line? I don't think we want to ship this project as a package.